SPECS-SIGNED/hvloader-signed/hvloader-signed.spec

```diff
@@ -6,7 +6,7 @@
 Summary:        Signed HvLoader.efi for %{buildarch} systems
 Name:           hvloader-signed-%{buildarch}
 Version:        1.0.1
-Release:        1%{?dist}
+Release:        2%{?dist}
```

Comment:
Ah, a first "signed" spec I haven't tested to run through fast-track. Fingers crossed.:)